static/app/utils/makeSafeRefluxStore.ts

```diff
@@ -0,0 +1,50 @@
+export interface SafeRefluxStore extends Reflux.Store {
+  teardown(): void;
+  unsubscribeListeners: Reflux.Subscription[];
```

Comment:
Should this be exposed on the public store interface?

Reply:
Yeah, we just need to cast to SafeStore instead of Reflux.Store... I will try and see if we can make this nicer somehow